content/scipy/concepts/scipy-signal/terms/find-peaks/find-peaks.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'find_peaks()'
+Description: 'Finds the peaks in a signal.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Visualization'
+Tags:
+  - 'Scikit-learn'
+  - 'Machine Learning'
+CatalogContent:
+  - 'paths/intermediate-machine-learning-skill-path'
+  - 'learn-python-3'
+---
+
+The **`find_peaks()`** function is used to find peaks in a signal. It returns the indices of local maxima in a signal.
+
+## Syntax
+
+```pseudo
+find_peaks(signal);
```

Comment:
Please correct the syntax. Here is the reference documentation: https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.find_peaks.html